test/tls/travis_test_client.py

```diff
@@ -25,18 +24,29 @@
 
 send_data = cipher_suite_code = version = None
 
-if len(sys.argv) >= 2:
-    send_data = sys.argv[1]
-
-if len(sys.argv) >= 3:
-    cipher_suite_code = int(sys.argv[2], 16)
-
-if len(sys.argv) >= 4:
-    version = int(sys.argv[3], 16)
-
-
-ch = TLSClientHello(version=version, ciphers=cipher_suite_code)
-t = TLSClientAutomaton(client_hello=ch, data=send_data)
-t.run()
-
-
+def run_tls_test_client(send_data=None, cipher_suite_code=None, version=None):
+    ch = TLSClientHello(version=int(version, 16), ciphers=int(cipher_suite_code, 16))
+    t = TLSClientAutomaton(client_hello=ch, data=send_data)
+    t.run()
+
+from travis_test_server import run_tls_test_server
+
+def test_tls_client(suite, version, q):
+    msg = "TestC_" + suite + "_data"
+    # Run server
+    q_ = multiprocessing.Manager().Queue()
+    th_ = multiprocessing.Process(target=run_tls_test_server, args=(msg, q_))
```

Comment:
Using a multiprocessing and it's wrapper allow us to kill the automaton when timed out